src/main/resources/application.yaml

```diff
@@ -79,8 +79,8 @@ spring:
 
 # Scheduler configuration
 scheduler:
-  camel-route-config: 0 0/45 * * * ?
-  camel-leaf-router-config: 0 0/30 * * * ?
+  camel-route-config: 0 0 22 * * ?
+  camel-leaf-router-config: 0 0 21 * * ?
```

Comment:
why cron expression not moved to az key vault ?...if that is in place  every time you need to trigger the build just restart the pods will automatically works